### PR TITLE
Unpackerr

### LINKED ISSUE #20: what if i dont use Deluge?
Hi, i am an Unraid user and i just saw your app listed in the store section.

I have been looking for something to unpack my Sonarr and radarr download automatically ever since i moved from a windows system, however i dont use Deluge, is there a way for this app to just talk to sonarr/radarr looking for downloads and unpack them into the same folder as the .rar files (i have been doing that manually) and then sonarr/radarr will automatically process them (or if this app could ping them to force a check that would be cool too!)
but if its only for deluge then I understand :)

### PR DESCRIPTION
**Pull request by davidnewhall**
Remove Deluge.

Closes #20.

More testing to do... Feedback welcomed!

### CODE REVIEW

**Comment by madcatsu** [comment]
Will this continue checking Deluge for download status? This seems like it will break if people have set their "completed" folder the same as their "downloads" folder in their BitTorrent client

**Comment by davidnewhall** [comment]
This is working for me right now! It requires passing in your download path - I updated the readme a little bit. I'll update more. It can be a confusing setup, but it definitely works. 

tl;dr: This app just got the download path from Deluge, so now it has to be provided in config.

**Comment by madcatsu** [comment]
Noted your documentation updates. How will this handle incomplete downloads? 

ie. I have 50 files from a RAR archive, 20 are done, including the base .rar file, and the other 30 are still being snatched.

What happens in this instance? How does Unpackerr know that all files for the torrent are complete?

**Comment by davidnewhall** [comment]
The `Logic` section in the readme may need an update? Let me know. 

Basically, this looks for `Status="Completed"` in the Sonarr and Radarr Queue Api (ie. Completed Download, Still Queued). Those only get to that state when the torrent client reports the download is finished.

**Comment by madcatsu** [comment]
All good, this makes sense and I must have glossed over that part and not understood correctly.

Let me know when you merge this PR and I'll update/rename my Alpine container 👍

**Comment by Nach0-Stallion** [comment]
So, im sure i have borked somthing, but since i started a radarr download i was getting a json error. i thought it was just because the download locally wasn't complete and still had temporary rar files but now they are all complete and ready to unpack
Annotation 2020-02-12 215947

**Comment by Nach0-Stallion** [comment]
after restarting unpackerr

unpackerr

**Comment by davidnewhall** [comment]
fixed above pasted errors

**Comment by davidnewhall** [comment]
@madcatsu new thing is out.

**Comment by madcatsu** [comment]
Got it. New container image over here https://hub.docker.com/repository/docker/madcatsu/unpackerr-alpine-daemon

**Comment by davidnewhall** [comment]
Added back to readme!
